packages/gitlab-backend/src/service/router.ts

```diff
@@ -21,12 +21,17 @@ function getBasePath(config: Config) {
 export interface RouterOptions {
     logger: Logger;
     config: Config;
+    secure?: boolean;
 }
 
 export async function createRouter(
     options: RouterOptions
 ): Promise<express.Router> {
-    const { logger, config } = options;
+    const {
+        logger,
+        config,
+        secure = process.env.NODE_TLS_REJECT_UNAUTHORIZED !== '0',
```

Comment:
This parameter should be added to the configuration. In this way you can get it in this way: 
```ts
config.getOptionalBoolean('gitlab.proxySecure');
```